### ISSUE TITLE
Issues with Nuka Ride 6.1 xVASynth voiceover by Axary

### ISSUE THREAD

**Issue by Kyrenaika** [iAmMe27/WoD#5]
[Wabbajack.current.log](https://github.com/iAmMe27/WoD/files/11197791/Wabbajack.current.log)
Wabbajack fails to download Nuka Ride 6.1 xVASynth voiceover by Axary.7z
I've already downloaded it manually, both as 7z and zip format.
It still attempts and fails to download it.
Since the files name is currently "Nuka Ride 6.1 Voices by Axary (xVASynth ) and Kalistara (11labs) 1.0.7z" I've also tried to rename it with no success.

**Comment by iAmMe27** [comment]
You need to download the latest update, that was released yesterday, to solve this.

**Comment by Kyrenaika** [comment]
Alrighty, then'll do a clean reinstall first too, just to be sure. thank you very much!
